tests/CLI/modules/file_tests.py

```diff
@@ -55,6 +55,26 @@ def test_volume_list(self):
             }],
             json.loads(result.output))
 
+    def test_volume_list_order(self):
+        result = self.run_command(['file', 'volume-list', '--order=1234567'])
+
+        self.assert_no_fail(result)
+        self.assertEqual([
```

Comment:
I dont like tests like this, they break too easily if the fixture is changed.

instead just do

json_result = json.loads(result.output)
self.assertEqual(json_result['id'], 100)
or something. If a few of the fields you expect to be there are correct, likely they rest are as well.

Reply:
Done.